src/lib/server/Config.cpp

```diff
@@ -1182,6 +1182,10 @@ Config::parseAction(ConfigReadContext& s,
 		action = new InputFilter::SwitchToScreenAction(m_events, screen);
 	}
 
+	else if (name == "toggleScreen") {
```

Comment:
Can we please use spaces for new code? Different viewers will indent tabs differently.

Reply:
I don't like tabs either, but this file use tabs. To match the whole file formatting, I use tab here.